### DISCUSSION TITLE
Where is the original EfficientNet Model?

### DISCUSSION THREAD

**Discussion by buttke** [nextcloud/recognize#815] [Q&A]
Hi, I'd like to use Recognize's EfficientNet V2 model (/models/efficientnetv2/) in my own project. The json file for the model suggests that it existed in another format previously. Is that version of the model available anywhere? I'd like to be able to run it using Keras. 

I tried converting it to the Keras format (`tensorflowjs_converter --input_format=tfjs_layers_model --output_format=keras`), but that failed (Expecting python dict contains `class_name` and `config` as keys).

**Comment by marcelklehr** [comment]
Hello @buttke 

You can find the original model at tfhub: https://tfhub.dev/google/collections/efficientnet_v2/1
